Base/ctkAppLauncher.cpp

```diff
@@ -647,6 +647,7 @@ bool ctkAppLauncherPrivate::readSettings(const QString& fileName, int settingsTy
 void ctkAppLauncherPrivate::runProcess()
 {
   this->Process.setProcessChannelMode(QProcess::ForwardedChannels);
+  this->Process.setInputChannelMode(QProcess::ForwardedInputChannel);
```

Comment:
since this function was introduced only in Qt 5.2, could you add a `#ifdef`

It currently causes the test to fail with:

```
/usr/src/AppLauncher/Base/ctkAppLauncher.cpp: In member function ‘void ctkAppLauncherPrivate::runProcess()’:
/usr/src/AppLauncher/Base/ctkAppLauncher.cpp:650:17: error: ‘class ctkInteractiveProcess’ has no member named ‘setInputChannelMode’
   this->Process.setInputChannelMode(QProcess::ForwardedInputChannel);
                 ^
/usr/src/AppLauncher/Base/ctkAppLauncher.cpp:650:37: error: ‘ForwardedInputChannel’ is not a member of ‘QProcess’
   this->Process.setInputChannelMode(QProcess::ForwardedInputChannel);
                                     ^
```

See https://doc.qt.io/qt-5/qprocess.html#setInputChannelMode

Something like this would do it:

```
#if (QT_VERSION >= QT_VERSION_CHECK(5, 2, 0))
  this->Process.setInputChannelMode(QProcess::ForwardedInputChannel);
#endif
```